index.js

```diff
@@ -37,6 +37,14 @@ module.exports = class customRPC extends Plugin {
 				small_text: this.settings.get('small_text', undefined) || undefined,
 			},
 		};
+
+		let buttons = [];
+		if (this.settings.get('button1', {label: "", url: ""}).label != "" && this.settings.get('button1', {label: "", url: ""}).url != "") buttons.push(this.settings.get('button1'));
```

Comment:
Why are these empty but in settings by default button 1 text is "powercord.dev" and the url is "https://powercord.dev"